tests/test_highlevel.py

```diff
@@ -901,3 +857,61 @@ def test_discrete(self):
         for truthy in [True, False, {}, None, "ser"]:
             rm = msprime.RecombinationMap.uniform_map(1, 0, discrete=truthy)
             self.assertEqual(rm.discrete, bool(truthy))
+
+    def test_zero_recombination_map(self):
```

Comment:
did you intend to include these zero recombination tests on this PR? totally cool if so

Reply:
Yeah, spotted they were in the wrong place and moved them to the right place. Documented in the commit message.